content/help/addons/hero-image.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Hero Image
+tags:
+- Addons
+- Hero Image
+- Image upload
+category: help
+menu:
+  help:
+    parent: addons
+    identifier: addon-hero-image
+versioning:
+  added: 2.6
+---
+
+## Overview
+
+{{% cloudfeature %}}
+
+This plugin provides an easy way to have a custom background image for every page of the forum. It allows you to upload this image in the dashboard and access it through a [Smarty Template](/developer/smarty/functions/#function-hero-image-url) or [Customize Theme](/help/appearance/custom-theme).
+
+It also allows you override an the main image on a per-category basis. Overriding this background image for a category will also override the background image for every category or discussion inside of that category. This allows you to have a tree of background images through your forum, managed through the dashboard.
```

Comment:
"or discussion"? Isn't it only per category? Do you mean that every discussion in that category will inherit it's category's image? If so phrase it that way.

I'm not crazy about the tree analogy. I think simply saying : 
"A category can optionally have its own background image. Any child categories inside this category will inherit from the parent category, unless it has its own image set.